### PR TITLE
Eqh 2988

### PR DESCRIPTION
**Pull request by jdbrown-USGS**
Browserified hazdev-accordion view.

### CODE REVIEW

File: gruntconfig/watch.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    tasks: [
+      'jshint:tests',
+      'browserify',
+      'mocha_phantomjs'
+    ]
+  },
+  scss: {
+    files: [
+      config.src + '/**/*.scss',
+      config.src + '/**/*.scss'
```
Comment (emcwhirter-usgs) [inline gruntconfig/watch.js]: I'm not positive, but I think you meant to do config.test on this 2nd line for the scss section.

**Comment by emcwhirter-usgs** [comment]
You should add ".build" to your .gitignore file, then un-commit your .build/ folder.

**Comment by emartinez-usgs** [comment]
- Building should not create a ```.build/example``` directory. Serve examples directly from ```example``` (top-level) directory.
- The ```.build/src``` folder should contain exactly 1 CSS (hazdev-accordion.css) and 1 JS (hazdev-accordion.js) file.
- Mocha should no longer be copied to ```.build/test``` directory. Serve from ```node_modules``` instead.
- The built ```.build/test/test.js``` code should not bundle source code. Instead, use the ```external``` configuration option while browserify-ing the code. The include a script tag in test.html that points to hazdev-accordion.js```.
- Example code should use simple CSS file(s) rather than SCSS. This is so there is no need to run any build steps on examples.
- No need for ```browserify:example``` target.
- No need for ```compass:example``` target.
- Update connect configuration.
  - Add livereload option to dev (any maybe dist) targets.
  - Add open open to auto-open pages when starting grunt.
- No need for ```copy:example``` or ```copy:mocah``` targets.
- No need for ```cssmin:example``` target.
- No need for htmlmin task.
- Rewrite uglify configuration to be more readable:
```
var uglify = {
  dist: {
    src: config.build + '/hazdev-accordion.js',
    dest: config.dist + '/hazdev-accordion.js'
};
```
- Add a reload target to watch task. Watch all files that directly affect what would be rendered in the browser and automatically reload when they change.
- Create the ```src/accordion``` folder. Move Accordion.js and accordion.scss into this folder.
  - Rename accordion.scss to Accordion.scss
  - Create ```src/hazdev-accordion.scss``` file that imports accordion/Accordion. Use this file as your compass ```specify``` target.
  - Update dependent tasks (browserify, compass, etc...) to reference code in correct location.
  - Update ```require``` statements to reference correct module location.
- Update Accordion.js implementation to align with the [coding style guide](https://gist.github.com/emartinez-usgs/75fc8d07cd360a5d4172).
- Remove .tmp from .gitignore file.

**Comment by emartinez-usgs** [comment]
- Remove unused dependencies from package.json
- SCSS imports should not include the .scss extention (see: hazdev-accordion.scss)
- No longer use ```Object.create({})```. Instead, create an anonymous object inline with all to-be attached properties declared and set to null; like so:
```
_this = {
  publicVarOne: null,
  publicVarTwo: null,

  publicMethodOne: null,
  publicMethodTwo: null
};
```

**Comment by emartinez-usgs** [comment]
- SCSS imports should only include module to import. i.e.:
```
@import 'accordion/Accordion';
```
- Example doesn't seem to work. May just be due to poor CSS import (above).

**Pull request merged**